molecule/default/molecule.yml

```diff
@@ -14,6 +14,8 @@ platforms:
     image: debian:8
```

Comment:
Even when Debian 8 should (not nescesarilly will) get security backports until July 2020. This branch no longer get security-updates since June 2018. Debian 9 is the `old stable` branch now so I wonder if we still ensure that `sb-debian-base` works under Debian 8. What do you think @sestrella @nickovivar ?

Reply:
@speedlight I'm not sure about dropping Debian 8 support yet since it still gets security updates `from 17th June 2018 to June 30, 2020`

https://wiki.debian.org/LTS

Reply:
@sestrella That's true but my concern is mainly that the updates aren't done by the Debian Security Team and important packages like mysql or nodejs (managed via apt) aren't no longer updated.

Reply:
Based on the conversation with the team we will keep support for Debian 8 for now.